src/app/modules/shared-components/components/page-navigator/page-navigator.component.ts

```diff
@@ -72,7 +72,7 @@ export class PageNavigatorComponent {
 
   getList$(watchedGroupId: string): Observable<NavMenuItem[]> {
     return this.itemNavigationService.getRootActivities(watchedGroupId).pipe(
-      map((navMenuRootItem: NavMenuRootItem) => navMenuRootItem.items)
+      map((navMenuRootItem: NavMenuRootItem) => navMenuRootItem.items.slice(0, 5))
```

Comment:
If it was just to take the first 5, calling the service with the "limit" parameter would be much more efficient!
But here, the request is " first the activity of the current group, then those from ancestor groups, and in any case not display more than 5 items.", so it is not just a slice anyway.

Reply:
Fixed